tests/tools/tools/serialize.fs

```diff
@@ -52,6 +52,7 @@ module SerializationTests =
     let testExplicitLayoutArray() = 
         let mutable p1 = new Point()
         p1.Rho <- 1.0f
+        let p1 = p1
```

Comment:
Won't work.  The assignment of p1.Rho requires p1 be mutable, Point is a structure.